lib/charms/router.py

```diff
@@ -37,6 +40,26 @@ def ip(*args):
 def _run(cmd, env=None):
     if isinstance(cmd, str):
         cmd = cmd.split() if ' ' in cmd else [cmd]
-    p = subprocess.Popen(cmd, env=env, stdout=subprocess.PIPE)
 
+    return ssh(cmd, config('vpe-router'), config('user'), config('pass'))
+
+    p = subprocess.Popen(cmd, env=env, stdout=subprocess.PIPE)
```

Comment:
We short circuit because it's only for proxy, but I didn't want to remove them since eventually we'll pull out the ssh stuff.